sources/Renderer/OpenGL/Profile/GLCore/OpenGLCore.h

```diff
@@ -21,6 +21,9 @@
 #   include <GL/gl.h>
 #   include <GL/glext.h>
 #   include <GL/glx.h>
+#   include <EGL/egl.h>
```

Comment:
This probably also needs to be wrapped in `LLGL_LINUX_ENABLE_WAYLAND`. Not everyone has EGL installed.